source/installation/restore-minio.rst

```diff
@@ -0,0 +1,274 @@
+
+.. _minio-restore-hardware-failure:
+
+==============================
+Recover after Hardware Failure
+==============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Distributed MinIO deployments rely on :ref:`Erasure Coding
+<minio-erasure-coding>` to provides built-in tolerance for multiple disk or node
+failures. Depending on the deployment topology, MinIO can tolerate the loss of
+up to half the drives or nodes in the deployment while maintaining read access
+to stored objects.
+
+While MinIO can operate in a degraded state without significant performance
+loss, administrators should replace failed hardware at the earliest convenience
+to mitigate the risk of more significant failures.
+
+The following table lists the typical types of failure in a MinIO deployment
+and links to procedures for recovering from each:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%
+
+   * - Failure Type
+     - Description
+
+   * - :ref:`Drive Failure <minio-restore-hardware-failure-drive>`
+     - MinIO supports hot-swapping failed drives with new healthy drives. 
+
+   * - :ref:`Node Failure <minio-restore-hardware-failure-node>`
+     - MinIO detects when a node rejoins the deployment and begins aggressively
+       healing data previously stored on that node.
+
+.. admonition:: MinIO Professional Support
+   :class: note
+
+   |subnet| provides 24/7 direct-to-engineering support for MinIO deployments.
+   SUBNET is a key component for ensuring smooth and reliable operation of MinIO
+   deployments, including root-cause analysis, health diagnostics, and
+   architecture reviews.
+
+   Existing SUBNET users should `log in <https://subnet.min.io/>`__ and
+   create a new issue to synchronize any recovery operations with MinIO
+   engineering.
+
+   Community users can seek support on the `MinIO Community Slack
+   <https://minio.slack.com>`__. Community Support is best-effort only and has
+   no SLAs around responsiveness.
+
+.. _minio-restore-hardware-failure-drive:
+
+Drive Failure Recovery
+----------------------
+
+MinIO supports hot-swapping failed drives with new healthy drives. MinIO
+detects and aggressively heals those drives without performing any restart
```

Comment:
Still needs a bit of tweaking imo.  The concern that needs to be addressed is not whether a restart is required, rather, will the heal negatively impact performance when a disk/disks/node/nodes are brought back online.

Reply:
Made additional tweaks